scrapers_next/ma/committees.py

```diff
@@ -0,0 +1,89 @@
+from spatula import URL, HtmlListPage, HtmlPage, CSS, XPath, selectors
+from openstates.models import ScrapeCommittee
+import requests
+
+requests.packages.urllib3.disable_warnings()
+
+
+class CommitteeDetail(HtmlPage):
+    def process_page(self):
+        com = self.input
+        com.add_source(
+            self.source.url,
+            note="Committee Details Page",
+        )
+        try:
+            member_blocks = XPath('//*[contains(@class, "committeeMemberList")]').match(
+                self.root
+            )[0]
+        except selectors.SelectorError:
```

Comment:
Change the import from `selectors` to `SelectorError`, because you are already directly importing other selectors like `CSS` and `XPath`.

Then this line will change to simply `except SelectorError:`